Kudu.Client/Infrastructure/HttpClientExtensions.cs

```diff
@@ -93,5 +93,22 @@ public static T GetJson<T>(this HttpClient client, string url)
 
             return JsonConvert.DeserializeObject<TOutput>(content);
         }
+
+        public static async Task<HttpResponseResult<TOutput>> RichPutJsonAsync<TInput, TOutput>(this HttpClient client, string url, TInput param)
```

Comment:
yep sure.. i was actually thinking about it at the very beginning, didn`t go with it was worry about people who call it might now know the feature. 

now look back seems like i was over-worry . 

will update it.